client/src/pages/OfferDash/index.tsx

```diff
@@ -0,0 +1,164 @@
+                    notes: [],
+                    events: [],
+                    contacts: []
+                },
+                {
+                    _id: '624e1b08284261a838b3b96a',
+                    company: 'Cvent',
+                    position: 'Software Engineer',
+                    location: 'Fordham, NC, USA',
+                    salary: 1000,
+                    cardColor: '#4285F4',
+                    progress: 1,
+                    jobPostUrl: 'https://careers.google.com/jobs/results/128310308272775878-software-engineer-iii/',
+                    description: `Google's software engineers develop the next-generation technologies that change how billions of users connect, explore, and interact with information and one another. Our products need to handle information at massive scale, and extend well beyond web search. We're looking for engineers who bring fresh ideas from all areas, including information retrieval, distributed computing, large-scale system design, networking and data storage, security, artificial intelligence, natural language processing, UI design and mobile; the list goes on and is growing every day. As a software engineer, you will work on a specific project critical to Google's needs with opportunities to switch teams and projects as you and our fast-paced business grow and evolve. We need our engineers to be versatile, display leadership qualities and be enthusiastic to take on new problems across the full-stack as we continue to push technology forward.`,
+                    notes: [],
+                    events: [],
+                    contacts: []
+                }
+            ]
+        };
+
+        // TODO: Make api call to get application info
+        setOfferData(cycle["applications"]);
+    }, []);
+
+    return (
+        <>
+            <Typography sx={{ fontWeight: 'bold', fontSize: h1Size, ml: 7, mt: 7 }} component="h1" variant="h4">
+                Offer Dashboard:
+            </Typography>
+            <Grid container sx={{ display: 'flex',  ml: 7, mt: 7 }}>
+                {offerData.map(offer => {
+                        return (
+                            <Grid sx={{ display: 'flex'}}> 
```

Comment:
Since you are using map, each child needs a key. Look at the console output on the frontend